VimeoNetworking/Sources/Request+Notifications.swift

```diff
@@ -8,22 +8,59 @@
 
 public extension Request
 {
-    private static var Path: String { return "/me/notifications/subscriptions" }
+    private typealias ParameterDictionary = [String: AnyObject]
+    
+    private static var SubscriptionsPathComponent: String { return "/subscriptions" }
     
     /// Retrieve the notification subscriptions.
     ///
     /// - Returns: subscriptionCollection
-    public static func getNotificationSubscription() -> Request
+    public static func getNotificationSubscription(notificationsURI: String) -> Request
     {
-        return Request(method: .GET, path: Path, parameters: nil)
+        let subscriptionsURI = notificationsURI.stringByAppendingString(SubscriptionsPathComponent)
```

Comment:
This URI isn't on the user model somewhere right? I'd be nice to not have to hardcode this